### PR TITLE
update: insecure deps

### PR DESCRIPTION
**Pull request by Velfi**
## Motivation and Context

aws-rust-sdk CI is failing on a cargo audit. This updates dependencies to avoid that.

## Description

update: insecure deps
add: extra context to timeout test failure
This doesn't fix all audit issues but the remaining ones are for examples so I think the path forward is to ignore them for now.

## Testing

ran existing tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/rust-runtime/aws-config/Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ aws-smithy-types = { path = "../../sdk/build/aws-sdk/sdk/aws-smithy-types" }
 aws-types = { path = "../../sdk/build/aws-sdk/sdk/aws-types" }
 tokio = { version = "1", features = ["sync"] }
 tracing = { version = "0.1" }
-hyper = { version = "0.14.16", default-features = false }
+hyper = { version = "0.14.13", default-features = false }
```
Comment (Velfi) [inline aws/rust-runtime/aws-config/Cargo.toml:26]:
```suggestion
hyper = { version = "0.14.16", default-features = false }
```

File: aws/sdk/integration-tests/s3control/Cargo.toml [hunk 2]
```diff
@@ -16,4 +16,4 @@ bytes = "1"
 http = "0.2.3"
 serde_json = "1"
 tokio = { version = "1", features = ["full", "test-util"] }
-tracing-subscriber = "0.2.18"
+tracing-subscriber = "0.3.5"
```
Comment (rcoh) [inline aws/sdk/integration-tests/s3control/Cargo.toml:19]: unless it's been fixed, upgrading to 0.3 requires adding the `env-filter` feature to be equivalent to 0.2

**Review by rcoh — COMMENTED** [review]
looks OK, but I think all the tracing subscribers need env-filter

**Review by jdisanti — APPROVED** [review]
LGTM!

**Pull request merged**